oct/ansible/oct/callback_plugins/log_results.py

```diff
@@ -1,16 +1,44 @@
 # coding=utf-8
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from codecs import open
-from datetime import datetime
-from os import environ, makedirs
-from os.path import exists, join, sep
-from shutil import rmtree
-from tempfile import mkdtemp
-from traceback import format_exc
+import contextlib
+from functools import wraps
+import logging
+import os
+from os.path import join
 
 from ansible.plugins.callback import CallbackBase
-from yaml import dump
+import structlog
+from structlog.stdlib import LoggerFactory
+
+
+structlog.configure(
+    processors=[
+        structlog.stdlib.add_log_level,
+        structlog.processors.TimeStamper(fmt="iso"),
+        structlog.processors.format_exc_info,
+        structlog.processors.UnicodeDecoder(),
+        structlog.processors.JSONRenderer(),
+    ],
+    context_class=dict,
+    logger_factory=structlog.stdlib.LoggerFactory(),
+    wrapper_class=structlog.stdlib.BoundLogger,
+    cache_logger_on_first_use=True)
+handler = logging.FileHandler(
+    join(os.environ.get('ANSIBLE_LOG_ROOT_PATH'), 'log.txt'), 'a')
```

Comment:
Single log file sounds good to me, but I think we still need that  environment variable to point to the containing directory. We set it in `oct/config/ansible_client.py` to communicate to the plugins where to put logs.